### PR TITLE
Games

### PR DESCRIPTION
**Pull request by SampooS**
Clicking on game-card now redirects to "reviews" while also saving clicked game id to local storage as "gameId".
also a bunch of small

### CODE REVIEW

**Review by Jihau — COMMENTED** [review]
Could you provide a screenshot of the page?

File: Server/src/interfaces/VerboseGame.ts [hunk 1]
```diff
@@ -11,6 +11,8 @@ interface VerboseGame {
   // the RAWG API id of the game.
   rating: number;
   // the rating of the game.
+  description: string;
+  // the description of the game.
```
Comment (mohammja) [inline Server/src/interfaces/VerboseGame.ts:14]: Make the documentation follow same system like this:
//documentation
code

**Review by mohammja — DISMISSED** [review]
Check the documentation of ur code

File: Server/src/interfaces/VerboseGame.ts (hunk 1 above)
Comment (SampooS) [reply Server/src/interfaces/VerboseGame.ts:14]: documentation formatting fixed, please re-check.

**Review by mohammja — APPROVED** [review]

**Comment by SampooS** [comment]
> Could you provide a screenshot of the page?
styling for the page isn't substantially changed, but here you go.
gamesgrid

File: Server/src/interfaces/VerboseGame.ts (hunk 1 above)
Comment (mohammja) [reply Server/src/interfaces/VerboseGame.ts:14]: good job!

**Pull request merged**